meta-lmp-base/files/lmp-group-table

```diff
@@ -44,6 +44,8 @@ pwm:x:54:
 games:x:60:
 render:x:61:
 weston-launch:x:62:
+weston:x:63:
+wayland:x:64:
```

Comment:
Do we need both groups here? Just because you're only adding weston user (and not making it part of the wayland group).

Reply:
Ricardo, I couldn't find another way to do it. I looked to the documentation, tried different ways, but anything different gave me errors like:

`weston-init was skipped: weston-init - weston-init: normal groupname weston does not have a static ID defined. Add weston to one of these files: /build/lmp/build-lmp-xwayland/conf/../../layers/meta-lmp/meta-lmp-base/files/lmp-group-table
`
and

`weston-init was skipped: weston-init - weston-init: system groupname wayland does not have a static ID defined. Add wayland to one of these files: /build/lmp/build-lmp-xwayland/conf/../../layers/meta-lmp/meta-lmp-base/files/lmp-group-table
`

Looking to the systemd-journal example, I found it similar to the weston-init recipe:

weston-init:
```
USERADD_PARAM_${PN} = "--home /home/weston --shell /bin/sh --user-group -G video,input weston"
GROUPADD_PARAM_${PN} = "-r wayland"
```
systems:
```
openembedded-core/meta/recipes-core/systemd/systemd_247.6.bb:397:USERADD_PARAM_${PN}-journal-gateway = "--system -d / -M --shell /sbin/nologin systemd-journal-gateway"
openembedded-core/meta/recipes-core/systemd/systemd_247.6.bb:398:USERADD_PARAM_${PN}-journal-remote = "--system -d / -M --shell /sbin/nologin systemd-journal-remote"
openembedded-core/meta/recipes-core/systemd/systemd_247.6.bb:399:USERADD_PARAM_${PN}-journal-upload = "--system -d / -M --shell /sbin/nologin systemd-journal-upload"

openembedded-core/meta/recipes-core/systemd/systemd_247.6.bb:389:GROUPADD_PARAM_${PN} = "-r systemd-journal;"
```